apex/optimizers/fused_adam.py

```diff
@@ -63,14 +63,17 @@ class FusedAdam(torch.optim.Optimizer):
     def __init__(self, params, lr=1e-3, bias_correction=True,
                  betas=(0.9, 0.999), eps=1e-8, adam_w_mode=True,
                  weight_decay=0., amsgrad=False, set_grad_none=True,
-                 capturable=False):
+                 capturable=False, use_master=False):
 
         if amsgrad:
             raise RuntimeError('FusedAdam does not support the AMSGrad variant.')
         # If the optimizer is capturable then LR should be a tensor (on GPU)
         lr = torch.tensor(lr, dtype=torch.float32) if capturable else lr
         defaults = dict(lr=lr, bias_correction=bias_correction,
                         betas=betas, eps=eps, weight_decay=weight_decay)
+        self.use_master = use_master
+        if self.use_master:
+            self.master_params = [p.data.clone().detach().float() for p in params]
```

Comment:
`params` can be a list of dicts (ref: https://pytorch.org/docs/stable/optim.html#torch.optim.Optimizer) so it might be worth referencing
- https://github.com/NVIDIA/apex/blob/7150e20cc3adef34e3f36261a1070fc0882f16a7/apex/optimizers/fused_mixed_precision_lamb.py#L46
- https://github.com/NVIDIA/apex/blob/7150e20cc3adef34e3f36261a1070fc0882f16a7/apex/optimizers/fused_mixed_precision_lamb.py#L117-L127
- https://github.com/NVIDIA/apex/blob/7150e20cc3adef34e3f36261a1070fc0882f16a7/apex/optimizers/fused_mixed_precision_lamb.py#L191